pytorch_toolkit/ote/ote/modules/evaluators/reid.py

```diff
@@ -42,9 +44,9 @@ def _evaluate_internal(self, config_path, snapshot, out, update_config, metrics_
         if os.path.islink(snapshot):
             snapshot = os.path.join(os.path.dirname(snapshot), os.readlink(snapshot))
 
-        update_config['classification.data_dir'] = Path(update_config['data.root']).name
-        update_config['data.root'] = Path(update_config['data.root']).parent
-        update_config = ' '.join([f'{k} {v}' for k, v in update_config.items() if len(str(v)) and len(str(k))])
+        data_path_args = f'--custom-roots {update_config[self.parameter_test_dir]} {update_config[self.parameter_test_dir]} --root _ '
```

Comment:
what does `--root _` mean?

Reply:
`root` actually is not required here, but it was added as a delimiter for argparse: if the last argument is a multiple-valued one (nargs=+), we can't fetch the remaining options: all of them are parsed as additional values. Now we have such arguments, so whey shouldn't be right before the remaining args